pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -224,14 +224,16 @@ func (b *OFBridge) createGroupWithType(id GroupIDType, groupType ofctrl.GroupTyp
 	return g
 }
 
-func (b *OFBridge) DeleteGroup(id GroupIDType) error {
+func (b *OFBridge) DeleteGroup(id GroupIDType, deleteFlows bool) error {
 	ofctrlGroup := b.ofSwitch.GetGroup(uint32(id))
 	if ofctrlGroup == nil {
 		return nil
 	}
-	g := &ofGroup{bridge: b, ofctrl: ofctrlGroup}
-	if err := g.Delete(); err != nil {
-		return fmt.Errorf("failed to delete the group: %w", err)
+	if deleteFlows {
```

Comment:
Could you add more comments or PR description? I didn't get that why the issue can be fixed by bypassing this part.

Reply:
updated